### ISSUE TITLE
[A11y] Insights Keyword Oregon Search Results Grid View

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3301]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Grid View
Link: https://oregondigital.org/catalog?locale=en&q%5B%5D=oregon&search_field=all_fields&utf8=%E2%9C%93&view=gallery
Keyword "Oregon"

Used Insights to validate page, contains 20 errors see screen snap below.

<img width="940" alt="Image" src="https://github.com/user-attachments/assets/094bdae7-3714-4ec1-9d3a-8fa5d03d21d1" />

### Suggested Fix

Run page using chrome insights, see. errors and suggested fixes. Example of suggested fix.

Fix the following:
Element is in tab order and does not have accessible text
Fix ONE of the following:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Re validate the page using chrome insights for errors.
